lib/commands.js

```diff
@@ -53,15 +69,33 @@ export default class Commands {
       messages.reverse()
     }
 
+    let found
+    let currentFileEncountered = false
     for (let i = 0, length = messages.length; i < length; i++) {
       const message = messages[i]
       const messageFile = $file(message)
       const messageRange = $range(message)
-      if (messageFile && messageRange && messageFile === currentFile && currentPosition.compare(messageRange.start) === expectedValue) {
-        visitMessage(message)
+
+      if (!currentFileEncountered && messageFile === currentFile) {
+        currentFileEncountered = true
+      }
+
+      if (messageFile && messageRange && (
+        (messageFile === currentFile && currentPosition.compare(messageRange.start) === expectedValue) || (currentFileEncountered && messageFile !== currentFile)
```

Comment:
The `if` conditionals are already split into multiple lines, might as well split this line too so you don't have to scroll horizontally to read it.